scripts/safe_template_linter.py

```diff
@@ -982,6 +1041,15 @@ def _check_javascript_file_is_safe(self, file_contents, results):
         self._check_concat_with_html(file_contents, results)
         self.underScoreLinter.check_underscore_file_is_safe(file_contents, results)
         results.prepare_results(file_contents)
+        self._filter_commented_code(results)
+
+    def _filter_commented_code(self, results):
+        """
+        Remove any results that were found in code that was commented out.
+        """
+        for violation in list(results.violations):
+            if violation.first_line().lstrip().startswith('//'):
+                results.violations.remove(violation)
```

Comment:
This code is a lot like the other implementation below (just the comment criterion is different).  Maybe refactor?  Also, it won't matter for short lists like this, but iterating and then using remove will be O(n**2).  You could do:
```
results.violations = [v for v in results.violations if not self.is_comment(v)]
```